package.json

```diff
@@ -72,6 +75,7 @@
     "react-native-app-auth": ">=6.4.3",
     "react-native-device-info": "^10.3.0",
     "react-native-keychain": "^8.1.1",
+    "react-native-root-toast": "^3.4.1",
```

Comment:
Did you consider https://www.npmjs.com/package/react-native-toast-message vs. this lib?

Reply:
I don't _think_ I tried this one. Maybe it is worth swapping out. I am getting some deprecated runtime warnings with the other library which I don't love.

Reply:
👍 the RootSiblingParent thing feels odd to me so I'm hopeful that other lib blows our minds.  It's published much more recently, has more weekly downloads, etc.  🤞